ANALYSIS/ANALYSISalice/AliAnalysisTaskMCParticleFilter.cxx

```diff
@@ -306,6 +309,7 @@ void AliAnalysisTaskMCParticleFilter::UserExec(Option_t */*option*/)
 	if(!hiH)hiH = dynamic_cast<AliGenHijingEventHeader*>(genHeaders->At(imch));
 	if(!colG)colG = dynamic_cast<AliCollisionGeometry *>(genHeaders->At(imch));
 	if(!dpmH)dpmH = dynamic_cast<AliGenDPMjetEventHeader*>(genHeaders->At(imch));
+  if(!hmcH)hmcH = dynamic_cast<AliGenHepMCEventHeader*>(genHeaders->At(imch));
```

Comment:
Is this indented properly?

Reply:
I think the file in general is not well indented because not all authors export tab to space. My editor is configured to export tab to space and therefore the indentation should be done properly. About the lines above I am not sure. I would avoid mixing reformatting and logical commits as the two would be hard to distinguish. However in general it would maybe make sense to reformat all classes in AliRoot to make the code a bit more readable - some files look awful!

The change here however is intended: I don't see why only Pythia export cross section, number of trials and pthard to the AOD MC header.